### ISSUE TITLE
Bug: flameshot setup

### ISSUE THREAD

**Issue by pascalbe-dev** [pascalbe-dev/ubuntu-os-setup#27]
Can not directly create screenshots with flameshot. Maybe the keybinding is wrong or the keybinding is blocked by the OS.
Maybe also flameshot is not part of the autostart.

**Comment by pascalbe-dev** [comment]
This is an issue in GNOME. A workaround can be found here:
https://github.com/flameshot-org/flameshot/issues/2594#issuecomment-1408247795
